crates/ln-dlc-node/src/ln_dlc_wallet.rs

```diff
@@ -91,6 +91,12 @@ impl LnDlcWallet {
 
         Ok(txs)
     }
+
+    pub fn get_new_address(&self) -> Result<Address> {
```

Comment:
🔧 It's a bit confusing that the method is named `get_new_address` when it actually returns an unused address.